src/components/dropdowns/base/index.js

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { TextField, MenuItem } from '@material-ui/core';
+import { TextField as FormikTextField } from 'formik-material-ui';
+import { Field } from 'formik';
+
+const GenericDropdownField = props => {
+  const { name, label, children, readOnly, ...otherProps } = props;
+
+  if (readOnly) {
+    const readOnlyProps = {
+      disableUnderline: true,
+      readOnly: true,
+    };
+
+    return (
+      <TextField label={label} InputProps={readOnlyProps} {...otherProps} />
```

Comment:
jw - how do we set the value for this?

Reply:
It's passed down in otherProps, so you can just set it.